CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+### citus v11.3.1 (February 12, 2024) ###
+
+* Fixes a bug related to non-existent objects in DDL commands (#6984)
+
+* Fixes a bug that could cause COPY logic to skip data in case of OOM (#7152)
+
+* Fixes a bug with deleting colocation groups (#6929)
+
+* Fixes incorrect results on fetching scrollable with hold cursors (#7014)
+
+* Fixes memory and memory context leaks in Foreign Constraint Graphs (#7236)
+
+* Fixes replicate reference tables task fail when user is superuser (#6930)
+
+* Fixes the incorrect column count after ALTER TABLE (#7379)
+
+* Improves citus_shard_sizes performance (#7051)
+
+* Makes sure to disallow creating a replicated distributed table
+  concurrently (#7219)
+
+* MERGE is unsupported with filters that prune down to zero shards (#6946)
```

Comment:
```suggestion
* Disallows MERGE when the query prunes down to zero shards (#6946)
```

And given this renaming, could you insert this entry into the correct place based on alphabetical ordering of the new items?

Reply:
Fixed